### PR TITLE
feat: adds "like all"

### LINKED ISSUE #206: Give anumodana to all
When I enter the page I usually like to give people anumodana. Right know I'm clicking on every single one. This isn't only sometimes a lot of "work", but also increases server load when people are doing this often. Every anumodana is a server round-trip which results in a broadcast websocket message and a refresh by everyone one the page.
What do you think about "Give anumodana to all"? 

@yuttadhammo @olastor

### PR DESCRIPTION
**Pull request by binarious**
Depends on https://github.com/Sirimangalo/meditation-plus-server/pull/33
Closes #206

---
This change is [Reviewable](https://reviewable.io/reviews/sirimangalo/meditation-plus-angular/228)

### CODE REVIEW

File: src/app/meditation/meditation.css [hunk 1]
```diff
@@ -2,6 +2,31 @@ md-card.nopadding {
...
 
+md-card-title {
+  display: flex;
+}
+
+.like-button {
+  width: 34px;
+  height: 34px;
+  font-size: 13px;
+  line-height: 13px;
```
Comment (olastor) [inline src/app/meditation/meditation.css]: Could you add a small `margin-right` (like 10px worked for me) to align the button vertically with the like number of a meditation in the list?

**Comment by binarious** [comment]
Review status: 0 of 7 files reviewed at latest revision, 1 unresolved discussion.

---

*[src/app/meditation/meditation.css, line 13 \[r1\]](https://reviewable.io:443/reviews/sirimangalo/meditation-plus-angular/228#-KQWdPgcS8gd9AzlX1bu-r1-13:-KQWxmZh4L-Fa9CDxK7P:b-896fix) ([raw file](https://github.com/sirimangalo/meditation-plus-angular/blob/47b8d3408851e74347a96c31c30b1730289f014e/src/app/meditation/meditation.css#L13)):*
<details><summary><i>Previously, olastor (Sebastian K.) wrote…</i></summary>
> Could you add a small `margin-right` (like 10px worked for me) to align the button vertically with the like number of a meditation in the list?
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/sirimangalo/meditation-plus-angular/228)*

File: src/app/meditation/meditation.css (hunk 1 above)
Comment (binarious) [inline src/app/meditation/meditation.css]: Done.

**Comment by olastor** [comment]
Reviewed 4 of 7 files at r1, 3 of 3 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/sirimangalo/meditation-plus-angular/228)*

**Pull request merged**